src/components/info-sheet/info-sheet.html

```diff
@@ -134,9 +134,8 @@
     </info-sheet-template>
 
     <info-sheet-template *ngSwitchCase="'copied-invoice-url'" type="{{ params.error ? 'danger' : 'success' }}">
-      <span *ngIf="!params.error" sheet-title-text
-        class="truncate">{{'Copied: {invoiceUrl}' | translate: params }}</span>
-      <span *ngIf="params.error" sheet-title-text class="truncate">{{'Error: {error}' | translate: params }}</span>
+      <span *ngIf="!params.error" sheet-title-text class="truncate" translate>Copied</span>: {{invoiceUrl}}
```

Comment:
Not sure about this. `*ngIf` doesn't include `:`.
This should be:
```
<span *ngIf="!params.error" sheet-title-text class="truncate">
  <span translate>Copied</span>: {{invoiceUrl}}
</span>
```
That makes sense?

Reply:
Ahh that makes sense thank you!